src/docfx/lib/UrlUtility.cs

```diff
@@ -348,6 +348,31 @@ public static string RemoveLeadingHostName(string url, string hostName, bool rem
                 : $"/{path}";
         }
 
+        public static string StandardizeBookmark(string uid)
+        {
+            uid = uid.ToLowerInvariant();
+
+            uid =
+                uid.
+                Replace("\"", "").
+                Replace("'", "").
+                Replace("%", "").
+                Replace("^", "").
+                Replace("\\", "").
+                Replace("<", "(").
+                Replace(">", ")").
+                Replace("{", "((").
+                Replace("}", "))");
+
+            uid = Regex.Replace(uid, @"[^a-zA-Z0-9()]", "-");
+
+            uid = Regex.Replace(uid, @"^-+", "");
+            uid = Regex.Replace(uid, @"-+$", "");
+            var bookmark = Regex.Replace(uid, @"-+", "-");
+
+            return bookmark.Length <= 100 ? bookmark : $"{bookmark.Substring(0, 100)}";
```

Comment:
remove unneeded string interpolation
```suggestion
            return bookmark.Length <= 100 ? bookmark : bookmark.Substring(0, 100);
```